ERCS/erc-x.md

```diff
@@ -0,0 +1,177 @@
+---
```

Comment:
```suggestion
---
eip: 7780
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.